interface/src/app/sample/services.js

```diff
@@ -0,0 +1,37 @@
+angular.module('adage.sample.services', [
+  'ngResource',
+  'adage.utils'
+])
+
+.factory('Sample', ['$resource', '$http', 'ApiBasePath',
+  function($resource, $http, ApiBasePath) {
+    var Sample = $resource(
+      ApiBasePath + 'sample/:id/',
+      // TODO need to add logic for handling pagination of results.
+      // then, can change "limit" below to something sensible
+      {id: '@id', limit: 0},
```

Comment:
Nice catch. This is one of the first services I wrote so I was probably following some sample code that I didn't fully understand. I will pull that part out and test.

- [ ] try simplifying `services.js` by removing `@id` from the `$resource` definition